### ISSUE TITLE
::get_term returns an array when passed a taxonomy name (string)

### ISSUE THREAD

**Issue by acobster** [timber/timber#2360]
# Expected behavior

I would expect, if we get an array back from `TermFactory`, that it would just return the first element. This is basically what `::get_post()` does.

```php
$term = Timber::get_term('my_tax'); // $term should be a Term or false
```

## Actual behavior

`$term` is an array instead.

## Steps to reproduce behavior

Just call `Timber::get_term('<taxonomy name>')`.

## What version of WordPress, PHP and Timber are you using?

Timber `2.x` branch (https://github.com/timber/timber/commit/d9ec0333ca0c81bf67ae3d4a3a0fff503e248bae at time of writing)

## How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)

N/A

**Comment by jarednova** [comment]
agreed!

**Comment by acobster** [comment]
Fixed in #2391
